libs/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019-2020 The STE||AR-Group
+# Copyright (c) 2020 The STE||AR-Group
```

Comment:
Let's keep this as it is as well.

Reply:
Yes I revert it on my other PR but I guess the script is actually changed on master, thanks!